packages/protocol/contracts/identity/Escrow.sol

```diff
@@ -33,6 +33,7 @@ contract Escrow is
 
   event Withdrawal(
     bytes32 indexed identifier,
+    // Note that `to` currently references the original sender of the payment
     address indexed to,
```

Comment:
this may also be a bug/unintended, but doesn't seem like there's a great way to fix this without breaking backwards compatibility?